scripts/tutorial_3.py

```diff
@@ -20,8 +20,16 @@ def main(args=None):
     spin_thread.start()
     interface.start_competition()
 
-    #TODO
-
+    # Turns on a debug topic to visualize bounding boxes and slots
+    # /ariac/sensors/display_bounding_boxes
+    interface.display_bounding_boxes = True
+    
+    looking_for_parts_in_bin = 2
+    while(1):
+        # publishes parts in the bin on /ariac/sensors/slot_occupancy
+        interface.get_bin_parts(looking_for_parts_in_bin)
```

Comment:
I don't think it is necessary for this method to publish to a topic. I would prefer if the function just returned a slot occupancy message and then you log the results.